tests/Makefile.am

```diff
@@ -1464,6 +1464,14 @@ TESTS += \
 	imfile-logrotate-copytruncate.sh \
 	imfile-logrotate-nocopytruncate.sh \
 	imfile-growing-file-id.sh \
+	imfile-ignore-old-file-0.sh \
+	imfile-ignore-old-file-1.sh \
+	imfile-ignore-old-file-2.sh \
+	imfile-ignore-old-file-3.sh \
+	imfile-ignore-old-file-4.sh \
+	imfile-ignore-old-file-5.sh \
+	imfile-ignore-old-file-6.sh \
+	imfile-ignore-old-file-7.sh
```

Comment:
There is a '\' missing at the end of the line. This causes the CI failure.

Reply:
ahh! thanks for the tip @rgerhards .